api_tests/tests/peers_using_ip.ts

```diff
@@ -30,38 +30,6 @@ async function assertPeersAvailable(actor: Actor, peers: Actor[]) {
 }
 
 describe("Peers using IP tests", () => {
-    it(
-        "alice-send-request-wrong-peer-id",
```

Comment:
This test actually doesn't need `Carol`, should we port it to using `twoActorTest`?

Reply:
AFAICT we have not defined the behaviour we want for peer ids for the new swap protocols. It is not possible to port this test because of that reason. Should we be defining this now? That question is the reason I elected to delete it, I don't think we should but I also don't think that that should block the removal of rfc003 because it is totally unrelated. rfc003 should not be used as a proxy todo list so lets delete this and track the issues somewhere and not rely on rfc003 code to show us what we still need to do.

Reply:
Tracking issue: https://github.com/comit-network/comit-rs/issues/2856